index.html

```diff
@@ -0,0 +1,216 @@
+                Don't just admire creations from others, get your hands dirty and create you own work using open source.
+              </p>
+            </li>
+
+            <li class="program__card flex flex-collumn flex-ai-c">
+              <span class="program__card-icon">
+                <i class="fa-solid fa-network-wired"></i>
+              </span>
+
+              <h3 class="program__card-title section-sub-title">ALC Connect</h3>
+
+              <p class="program__card-description">
+                Rub shoulders and link up with ALC affiliates from all around the world during and after the conference.
+              </p>
+            </li>
+          </ul>
+
+          <a href="/pages/about.html" class="program__link">SEE THE WHOLE PROGRAM</a>
+
+          <div class="program__cta">
+            <span>
+              <a href="" class="program__cta-button"> Join ALT Conferece 2023 </a>
+            </span>
+          </div>
+        </div>
+      </section>
+
+      <section id="speaker" class="speaker section">
+        <div class="container container-pall">
+          <h2 class="section-title">Speakers</h2>
+
+          <ul class="speaker__grid"></ul>
+
+          <div class="speaker__cta">
+            <a class="speaker__cta-button speaker__expand flex flex-jc-c flex-ai-c">
+              <span>More</span> <i class="fa-solid fa-angle-down"></i>
+            </a>
+
+            <a href="#speaker" class="speaker__cta-button speaker__minimize flex flex-jc-c flex-ai-c">
+              <span>Less</span> <i class="fa-solid fa-angle-up"></i>
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section id="partners" class="partner section">
+        <div class="container container-pall">
+          <h2 class="partner__title section-title">Partners</h2>
+          <div class="partner__grid flex flex-jc-c flex-ai-c flex-wrap">
+            <img src="images/mozilla-logo.svg" alt="Mozilla logo" class="partner__logo" />
+
+            <img src="images/google-log.svg" alt="Google logo" class="partner__logo" />
+
+            <img src="images/naver-logo.svg" alt="Naver logo" class="partner__logo" />
+
+            <img src="images/daumkakao-logo.svg" alt="Daumkakao logo" class="partner__logo" />
+
+            <img src="images/airbnb-logo.svg " alt="Airbnb logo" class="partner__logo" />
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer section">
+      <div class="container container-pall flex flex-collumn flec-jc-c flex-ai-c">
+        <img class="footer__logo" src="images/logo-dark.svg" alt="logo" />
+
+        <div class="footer__info">
+          <p class="footer__copyright">2023 Africa's Leading Tech Conference. Some rights reserved</p>
+          <p class="footer__credits">
+            <a class="about__link" href="https://github.com/Timbar09" target="_blank">Credits</a>
+          </p>
+        </div>
+      </div>
+    </footer>
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/210952414-af0f8062-ddd5-40bc-b98b-062fc243725d.png)
- as required your text should have a logo aligned to the left and text aligned to the right  😄 
- your background colour should be the same as the background colour of your home section

Reply:
Thanks for the feedback @alexander16108 

I don't understand what I should change here. Can you please show me an example?